packages/smooth_app/lib/pages/user_preferences_dev_mode.dart

```diff
@@ -335,6 +337,36 @@ class UserPreferencesDevMode extends AbstractUserPreferences {
             setState(() {});
           },
         ),
+        ListTile(
+          leading: const Icon(Icons.language),
+          title: DropdownButton<String>(
+            value: userPreferences.appLanguageCode ??
+                Localizations.localeOf(context).toString(),
+            elevation: 16,
+            isExpanded: true,
+            onChanged: (String? languageCode) async {
+              await userPreferences.setAppLanguageCode(languageCode!);
+              setState(() {});
+            },
+            items: AppLocalizations.supportedLocales.map((Locale locale) {
+              final String _locale = locale.toString();
+              return DropdownMenuItem<String>(
+                value: _locale,
+                child: Text(_locale),
+              );
+            }).toList(),
+          ),
+        ),
+        ListTile(
+          title: Text(
+            appLocalizations.dev_preferences_reset_app_language,
+          ),
+          onTap: () async {
+            await userPreferences
+                .setAppLanguageCode(Platform.localeName.substring(0, 2));
```

Comment:
I was more thinking of setting it to `null` or completely removing the value if possible so that the standard value is used in main.dart

Reply:
Fixed. Please check it if you have time.